engine/runtime-language-epb/src/main/java/org/enso/interpreter/epb/PyForeignNode.java

```diff
@@ -21,10 +18,7 @@
 import com.oracle.truffle.api.library.ExportMessage;
 import com.oracle.truffle.api.source.Source;
 
-@NodeField(name = "foreignFunction", type = Object.class)
-public abstract class PyForeignNode extends ForeignFunctionCallNode {
-  @Child
-  private CoercePrimitiveNode coercePrimitiveNode = CoercePrimitiveNode.build();
+final class PyForeignNode extends GenericForeignNode {
```

Comment:
Python integration can in general reuse `GenericForeignNode`, but it has to do additional conversions:
- reported as #7655
- improved as https://github.com/oracle/graalpython/issues/354
- another reported as https://github.com/oracle/graalpython/issues/353
- and others

New versions of GraalPy shall (incrementally) remove the need for these conversions.